### DISCUSSION TITLE
Discrepancy between Zone alerts and County Alerts

### DISCUSSION THREAD

**Discussion by michael-digi** [weather-gov/api#418] [Q&A] [answered]
Hi all,

I may be unaware of some way the API works, but yesterday something odd happened. There were thunderstorms throughout MD that covered a large area and were getting severe, but no alerts for MDZ zones were appearing anywhere.

However, when I searched for alerts for MDC (via the Maryland county codes), it showed all the alerts I was expecting (severe thunderstorm warnings etc).

An example using Washington County MD here (by the time anyone looks here they may all be gone or the issue may not be presenting itself):

[MDC043](https://api.weather.gov/alerts?zone=MDC043)

[MDZ003](https://api.weather.gov/alerts?zone=MDZ003)

This county/zone combo overlaps but results only return when I search the county. Is this an issue on the API side or is this something I have to prepare/account for? Thanks for any insight.

-Michael

**Comment by JJKraw** [comment, marked as answer]
I've found that everything related to the Storm Prediction Center (SPC), like tornado and severe thunderstorm watches/warnings, are issued on the county level only. The warnings will also contain a polygon that defines the precise warned area. This pre-dates the API.

**Reply by michael-digi** [reply]
I appreciate the clarification, this changes how I was thinking about the system a bit. One last question, does this mean certain types of events by their very nature are only sent out as zone alerts vs county alerts? Like there are some events so large (thunderstorms, tornado watches) that they are only sent out as county alerts, and some events are so small they are sent via zone alerts?

**Comment by JJKraw** [comment]
I can't give you an authoritative answer, but from what I've seen it's only tornado and severe tstorm alerts issued with only county codes. Not really based on size (i.e. winter storm alerts can cover larger areas but are always issued with the zone codes).

Note that a point fetch of the form "https://api.weather.gov/alerts?active=1&point={lat},{long}" will give you both. There is an outstanding bug, however, because fire zone alerts aren't included. It is known and being worked on (INC0234358).

**Reply by jfbauer432** [reply]
I have been using "https://api.weather.gov/alerts/active?status=actual&point={lat},{long}" and in addition to the the usual storm warnings/watches/advisories/statements, I have received some fire related alerts: the type where conditions are good (or is that bad) for fires.  i.e. windy and low humidity.  I never paid close enough attention to know if they were zone or county based.

**Reply by JJKraw** [reply]
Might be worthwhile next time to note if the forecast zone code is the same as the fire zone code. The cases I've seen are where the fire zones don't match up one to one with the forecast zones (e.g., places in California).

**Reply by JJKraw** [reply]
I got an email yesterday saying the fix for returning fire zone alerts on point requests has been verified on the development system and will hit production in July.
